dist/systemd/system/greenboot-rollback.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Greenboot rollback
+DefaultDependencies=no
+Before=systemd-update-done.service
+After=local-fs.target
+ConditionNeedsUpdate=|/var
+ConditionNeedsUpdate=|/etc
+Conflicts=shutdown.target
+Before=shutdown.target
+
+[Service]
+Type=oneshot
+RemainAfterExit=true
+ExecStart=/usr/libexec/greenboot/greenboot rollback
```

Comment:
There was a discussion about unit ordering here because this currently invokes `rpm-ostree rollback` which is going to require dbus.service at least to be up and `/boot` to be mounted.

We could avoid the first part by using `ostree` directly (or the underlying ostree APIs).

But bigger picture is the goal here to ensure that higher level services (e.g. in a kubernetes case, `kubelet.service`) aren't started if we're going to roll back?

There are a few options for this, such as defining an entirely new target to boot into that has only the specified services enabled.

Or, instead the relevant services can be ordered after a service; this is the role that `boot-complete.target` serves in https://systemd.io/AUTOMATIC_BOOT_ASSESSMENT/